app/src/main/java/org/schabi/newpipe/player/notification/NotificationUtil.java

```diff
@@ -99,10 +99,7 @@ private synchronized NotificationCompat.Builder createNotification() {
         // build the compact slot indices array (need code to convert from Integer... because Java)
         final List<Integer> compactSlotList = NotificationConstants.getCompactSlotsFromPreferences(
                 player.getContext(), player.getPrefs(), nonNothingSlotCount);
-        final int[] compactSlots = new int[compactSlotList.size()];
-        for (int i = 0; i < compactSlotList.size(); i++) {
-            compactSlots[i] = compactSlotList.get(i);
-        }
+        final int[] compactSlots = compactSlotList.stream().mapToInt(i -> i).toArray();
```

Comment:
Reference for why this is recommended?

Reply:
https://stackoverflow.com/a/23945015

Reply:
I think `.mapToInt(Integer::intValue)` is clearer